Formula/mypaint-brushes.rb

```diff
@@ -0,0 +1,29 @@
+class MypaintBrushes < Formula
+  desc "Brushes used by MyPaint and other software using libmypaint"
+  homepage "https://github.com/mypaint/mypaint-brushes"
+  url "https://github.com/mypaint/mypaint-brushes/archive/v1.3.0.tar.gz"
+  sha256 "704bb6420e65085acfd7a61d6050e96b0395c5eab078433f11406c355f16b214"
+  license "CC0-1.0"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  depends_on "libmypaint"
+
+  def install
+    ENV["ACLOCAL"] = "aclocal"
+    ENV["AUTOMAKE"] = "automake"
+
+    system "./autogen.sh"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    # This cop produces a suggestion which does not work, as String does not have an exist? method
+    # https://github.com/yugabyte/linuxbrew/blob/master/Library/Homebrew/test/rubocops/lines_cop_spec.rb#L130
+    result = File.exist?("#{share}/mypaint-data/1.0/brushes/classic/marker_small_prev.png")
+    assert result
```

Comment:
```suggestion
    assert_predicate share/"mypaint-data/1.0/brushes/classic/marker_small_prev.png", :exist?
```
Is there a way to test these brushes other than checking they exist?

Reply:
All the package does is copy them into place. So there isn't really a better way to test them.

Reply:
From [here](https://github.com/mypaint/mypaint-brushes/blob/v1.3.x/configure.ac#L17-L21)

```
# The data version indicates data compatibility.
# Basically this is part of 1.0 series of data which will work with
# libmypaint 1.x. It is not incremented with package versions unless
# incompatible changes in data format are made.
```

Which then sets: `@MYPAINT_DATA_VERSION@` in `[pkgconfig.pc.in](https://github.com/mypaint/mypaint-brushes/blob/master/pkgconfig.pc.in)`

There is a 2.0 of the data format and the package is at version 2.0.2 on `master` but that version is for compatibility with `libmypaint` 2.0 which is not released.

Reply:
Done in 7955cea